From 2c46f8a096f96a359afef46860c884fc5b342ae3 Mon Sep 17 00:00:00 2001
From: Vidya Sagar Velumuri <vvelumuri@marvell.com>
Date: Fri, 26 Apr 2024 00:34:01 -0700
Subject: [PATCH 318/513] marvell-ci: reduce the crypto perf numbers for 96

Update the crypto perf numbers for 96

Signed-off-by: Vidya Sagar Velumuri <vvelumuri@marvell.com>
Change-Id: I13cb4f3c36e16fd631fa58e7fd9b050d6d25eb03
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/126147
Reviewed-by: Anoob Joseph <anoobj@marvell.com>
Tested-by: Anoob Joseph <anoobj@marvell.com>
(cherry picked from commit 29b5f76a58744684428bac56ef9aed60969412f7)
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/126270
---
 .../ref_numbers/cn9k/rclk2200_sclk1100_cptclk1000.96xx          | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/marvell-ci/test/cnxk-tests/crypto_perf/ref_numbers/cn9k/rclk2200_sclk1100_cptclk1000.96xx b/marvell-ci/test/cnxk-tests/crypto_perf/ref_numbers/cn9k/rclk2200_sclk1100_cptclk1000.96xx
index 18fea7922c2e3..66e4610e3616e 100644
--- a/marvell-ci/test/cnxk-tests/crypto_perf/ref_numbers/cn9k/rclk2200_sclk1100_cptclk1000.96xx
+++ b/marvell-ci/test/cnxk-tests/crypto_perf/ref_numbers/cn9k/rclk2200_sclk1100_cptclk1000.96xx
@@ -16,7 +16,7 @@ aes-gcm
 aes-cbc_sha1-hmac-ipsec
 64: 11.267 28.718 36.796
 384: 13.050 25.210 26.604
-1504: 11.163 12.921 11.880
+1504: 10.569 11.110 11.880
 <end>
 aes-gcm-ipsec
 64: 16.938 27.891 44.529
-- 
2.25.1

